app/view/mod_job_list.R

```diff
@@ -48,25 +53,9 @@ server <- function(id, state) {
 
     output$job_list_table <- renderReactable({
 
-      processed_jobs <- job_list_data() %>%
-        select(id, key, start_time, end_time) %>%
-        mutate(
-          job = paste(
-            id,
-            key,
-            start_time,
-            end_time,
-            sep = "_-_"
-          )
-        ) %>%
-        select(
-          -c(
-            id,
-            key,
-            start_time,
-            end_time
-          )
-        )
```

Comment:
Removing of this part should also affect the `box::use` statement. You can check the new `box.linters` package to get some details.